src/customer/strategies/amazon-pay-v2/amazon-pay-v2-customer-strategy.ts

```diff
@@ -63,6 +63,12 @@ export default class AmazonPayV2CustomerStrategy implements CustomerStrategy {
         );
     }
 
+    executePaymentProviderCheckout(options?: ExecutePaymentProviderCheckoutOptions): Promise<InternalCheckoutSelectors> {
```

Comment:
Just want to understand if this is part of the scope? Since this task is for `Bolt`.

Reply:
Yes, it is a new generic method for all providers customer strategies.

Reply:
We can't build this kind of feature only for bolt, because we should have an ability to use the feature for another payment providers.

Reply:
Thanks for the explanation. Nice work on doing that and can see the thinking of how we can use it for other providers too.